packages-exp/auth-exp/src/core/action_code_url.ts

```diff
@@ -45,11 +49,20 @@ const MODE_TO_OPERATION_MAP: { [key: string]: externs.Operation } = {
 
 /**
  * Maps the mode string in action code URL to Action Code Info operation.
+ *
+ * @param mode
+ * @internal
```

Comment:
You don't need to do this right? Since these are not exported?

Reply:
I think you're right, doing it for consistency here anyway, no promises I'll do it everywhere